### DISCUSSION TITLE
🚀 Flex Grow in Loop Grid - Could Be Bug 🐞

### DISCUSSION THREAD

**Discussion by awakekat** [elementor/elementor#22992] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When I manually create inside of a container in a column: Image, Title, Text Editor and a Button (top to bottom). I duplicate 2 more containers with the same content but I vary the length of the text in the text editor to make the button higher and lower from the bottom of the container. I use the flexbox grow feature on the text editor and I am able to move the button to the end of the container to fill the gap between the text and the button so that the button is flush to the bottom of the container. YAY!
However when I try to do the same thing inside of a loop grid

### Describe the solution you'd like

It would be nice if the Flexbox  grow feature worked inside of a loop grid to push the button down to align to the bottom of the container.

### Additional context

https://youtu.be/l7jLjoFC888

**Comment by ThrudUK** [comment]
I'm seeing something similar as well, a slightly different use case to get 2 side by side containers to be the same height. Works just fine if I build it manually, but why would I want to do that when I am inserting CPT data!
